esy-configure

```diff
@@ -18,15 +18,14 @@
 
 set -u
 set -e
-set -o pipefail
 
 case "$(uname -s)" in
     CYGWIN*|MINGW32*|MSYS*)
         echo "[esy-configure] Detected windows environment..."
-        ./configure-windows "$@"
+        exec ./configure-windows "$@"
```

Comment:
I intenter to use `.` instead of `exec` (so it inherit the -u and -e flag). But this seem to be an issue with POSIX sh because it is unspecified. If a FreeBSD guy can say if it is supported. Because . is better than exec.

Reply:
@ulrikstrid Can you weigh in here?

Reply:
Now I see this change the script become very slim and maybe we can just inline it ? so we get -e and -u propagated and still avoid one creation process.

Reply:
This is correct, exec is the right stuff here. My tip if you want to try if something is posix you can try on ash.